cdap-ui/app/cdap/components/DataPrep/Directives/Filter/index.js

```diff
@@ -137,7 +137,7 @@ export default class FilterDirective extends Component {
         directive = `${condition} ${column} ${textValue}`;
         break;
       case 'CUSTOMCONDITION':
-        directive = `${condition} ${this.state.customFilter}`;
+        directive = `${condition} ${column} ${this.state.customFilter}`;
```

Comment:
Yeah, because now the `this.state.customFilter` doesn't include the column name anymore. 

Relevant comment: https://issues.cask.co/browse/CDAP-9208?focusedCommentId=29046&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-29046